server/Dockerfile

```diff
@@ -20,4 +20,4 @@ RUN chown -R node:node /app
 USER node
 
 EXPOSE ${DB_HOST}
-CMD ["npm", "run", "dev"]
+CMD ["npm", "run", "docker"]
```

Comment:
Sorry, I should've been more clear but I believe Heroku runs whatever command is specified in the Dockerfile so if the `docker` script sets `DB_LOCAL=true`, then our db connection config will likely fail since we'd be using the wrong config. Do you know if there's a way to run different commands to start the server, based on the environment, in either the docker-compose file or Dockerfile?

Reply:
Ohh got it. I'll look into it, there should be a simple way to do that!

Reply:
I'm sure there's a better way to handle this, but I've created `Dockerfile.local` for now so that my `docker-compose.yml` file is looking at that and the original `Dockerfile` stays the same. I'm sure how the production build has been set up on the Heroku end, would this hopefully work for our current workflow?

Reply:
I think this is fine and shouldn't affect production build. But @jacksonyuanjx can you confirm?

Reply:
Hmm, I think this'll also work but it's hard to say as I'm not completely certain whether Heroku looks at the docker-compose file first or only at the Dockerfile. But let's try this and if it doesn't work we can always tweak things later.

Reply:
Thanks for going over this! Hopefully things go smoothly, but ping me if the prod happens to go down because of this change!